src/codegen/codegen_info.cpp

```diff
@@ -105,5 +107,24 @@ bool CodegenInfo::nrn_state_has_eigen_solver_block() const {
                 .empty();
 }
 
+/**
+ * Check if WatchStatement uses voltage variable v
+ *
+ * Watch statement has condition expression which could use voltage
+ * variable `v`. To avoid memory access into voltage array we check
+ * if `v` is used and then print necessary code.
+ *
+ * @return true if voltage variable b is used otherwise false
```

Comment:
After CostVisitor PR from Tristan we're using `\return` to comply with the latest coding conventions, so maybe we can already do this here like this
```suggestion
 * \return true if voltage variable b is used otherwise false
```